install.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+echo "Creating venv..."
+mkdir venv
+python -m venv venv
+echo "Activating venv..."
+source ./venv/Scripts/activate
+echo "Installing requirements..."
+pip install -r requirements.txt
+echo "Successfully installed!"
```

Comment:
This might get better if there was a prompt to ask if the user wants to download a model, and suggest some of the common models and download with curl or equivalent (apparently curl is available on Windows 10 too, see https://techcommunity.microsoft.com/t5/containers/tar-and-curl-come-to-windows/ba-p/382409)

Reply:
I tend to agree about asking. Sometimes people are on expensive connections and should have the chance to put off large downloads until it costs less or nothing.